doc/modules/model_evaluation.rst

```diff
@@ -806,6 +810,18 @@ precision-recall curve as follows.
         :scale: 75
         :align: center
 
+Precision can also be referred to as the `positive predictive value (PPV)
```

Comment:
I am thinking that we could isolate these two metrics in a new section where we can provide more details regarding binary and multiclass case and the effect of the averaging similarly to the precision-recall section

Reply:
Attempted.